demos/index.html

```diff
@@ -59,6 +59,13 @@
               <span class="mdc-list-item__secondary-text">Multi-selection controls</span>
             </a>
           </li>
+          <li class="mdc-list-item">
```

Comment:
Do we really need a demo page for both context menus and simple menu? I think the updates you did to the simple menu demo page show the anchor positioning better than the context menu demo...so I kinda just want to drop the entire context menu demo

Reply:
ok